src/components/manage/Blocks/Video/Edit.jsx

```diff
@@ -253,6 +253,15 @@ class Edit extends Component {
                             : data.url
                         }
                         controls
+                        poster={
+                          data.preview_image
+                            ? isInternalURL(data.preview_image)
+                              ? `${flattenToAppURL(
+                                  data.preview_image,
+                                )}/@@images/image`
```

Comment:
This is wrong, right?

Reply:
No, We are using it everywhere.
https://github.com/plone/volto/blob/master/src/components/manage/Blocks/Image/Edit.jsx#L268

Reply:
For large image